src/executables/util_service.c

```diff
@@ -5176,13 +5186,15 @@ process_heartbeat (int command_type, int argc, const char **argv)
     case RELOAD:
       status = process_heartbeat_reload (argc, argv);
       break;
+#if defined (ENABLE_UNUSED_FUNCTION)
```

Comment:
I don't really know, but in this function, there is a path executing copylogdb/applylogdb. Are they blocked?

process_heartbeat() -> us_hb_process_start() -> us_hb_copylogdb/applylogdb_start()

Reply:
oh.. they are removed in #4364.